source/includes/steps-copy-cli.yaml

```
@@ -0,0 +1,114 @@
+content: |
+  Create a new {+app+}. This is where you'll add the configuration files from 
+  the exported app in later steps. To create the app, run: 
+
+  .. code-block:: bash
+     
+     realm-cli app create
+
+  Follow the CLI prompts to complete app creation. This creates a new directory on 
+  your local machine with the new app's config files.
+
+---
+title: Copy App Configuration to New App
+ref: copy-app-cli-copy-exported-app-to-new-app
+content: |
+  Delete all the configuration files in the new app except for ``realm_config.json``.
+  You'll handle ``realm_config.json`` in the next step. 
+
+  Copy all the :ref:`configuration files and folders <app-configuration>` of your exported app
+  to the imported app folder except for the ``realm_config.json`` file.
+
+---
+title: Configure the New App's ``realm_config.json``
+ref: copy-app-cli-configure-realm-config-json
+content: |
+  In the new app's ``realm_config.json``, replace all the contents except for the 
+  ``"app_id"`` and ``"name"``.
+
+  The new app's ``realm_config.json`` should look something like: 
+
+  .. code-block:: javascript
+
+     {
+       "app_id": "mynewapp-abcde", // don't change
+       "name": "MyNewApp", // don't change
+       "config_version": 20210101, // copy from exported app
+       "location": "US-VA", // copy from exported app
+       "deployment_model": "GLOBAL" // copy from exported app
+     }
+
+---
+title: Migrate Secrets to New App
+ref: copy-app-cli-migrate-secrets
+content: |
+  When you :ref:`export your {+app+} <export-realm-app>`, the export does not include the app's 
+  :ref:`secrets <app-secret>`. As secrets cannot be read once they're defined, 
+  you must also have access to the secrets outside of {+service-short+} to import 
+  them into a {+app+}. If your app doesn't have any secrets, you can skip this step.
+
+  To add the secrets from another Realm app:
+
+  #. Get the names of all the secrets from the exported app by following the 
+     :ref:`view secrets documentation <list-secrets>`.
+  #. Save the names of all the secrets to a secure location. You won't have access 
+     to the actual secrets, but it'll be useful to have a list of the secret names 
+     to add to your imported app.
+  #. Gather the secrets. This cannot be done from within {+service-short+}. 
+  #. Add all the secrets to the imported app by following the :ref:`define a secret documentation <define-secret>`.
+     Secrets must be added one at a time.
+
+  .. note:: Add secrets before pushing app
+
+     You should add all secrets before puhsing up the copy of the {+app+} because 
+     {+service-short+} can require that the secrets exist to push the configuration.
+     If the secrets don't exist, the {+cli+} will throw an error when you push the 
+     app in the next step.
+
+---
+title: Push New App to Realm
+ref: copy-app-cli-push-to-realm
+content: |
+  Now that the app configuration and secrets are updated, push configuration to your
```

Reads slightly odd to me without an article.

```suggestion
  Now that the app configuration and secrets are updated, push the configuration to your
```